frame/im-online/src/mock.rs

```diff
@@ -156,15 +159,15 @@ parameter_types! {
 }
 
 impl pallet_session::Config for Runtime {
-	type ShouldEndSession = pallet_session::PeriodicSessions<Period, Offset>;
+	type ShouldEndSession = pallet_session::PeriodicSessions<ConstU64<1>, ConstU64<0>>;
```

Comment:
```suggestion
	type ShouldEndSession = pallet_session::PeriodicSessions<Period, Offset>;
```
If we continue to define those constant and use it it makes no sense to duplicate them